src/Dto/KeycloakUserValidateDto.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Dto;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class KeycloakUserValidateDto {
+    #[Assert\NotBlank]
+    public string $domain;
```

Comment:
Would make the properties private and use getter and setter. We do this in other parts of the application in that way.